osu.Framework/Graphics/Shaders/ShaderManager.cs

```diff
@@ -83,14 +83,16 @@ private IShaderPart resolveShaderPart(string name, ShaderPartType type)
             byte[]? rawData = GetRawData(name);
 
             if (rawData == null)
-                throw new FileNotFoundException($"{type} shader part could not be found.", name);
+                throw new FileNotFoundException($"{type} shader part could not be found. If using a different file extension than .fs or .vs, Be sure to include it in the argument.", name);
```

Comment:
This doesn't need to be changed. The full filename is already included in the exception output which should let a developer understand the issue.

```suggestion
                throw new FileNotFoundException($"{type} shader part could not be found.", name);
```